cmd/bb_storage/main.go

```diff
@@ -40,6 +52,32 @@ func main() {
 		log.Fatal(http.ListenAndServe(*webListenAddress, nil))
 	}()
 
+	pe, err := prometheus_exporter.NewExporter(prometheus_exporter.Options{
+		Namespace: "bb_storage",
+		Registry: prometheus.DefaultRegisterer.(*prometheus.Registry),
+	})
+	if err != nil {
+		log.Fatalf("Failed to create the Prometheus stats exporter: %v", err)
+	}
+	view.RegisterExporter(pe)
+	if err := view.Register(ocgrpc.DefaultServerViews...); err != nil {
+		log.Fatalf("Failed to register ocgrpc server views: %v", err)
+	}
+	zpages.Handle(nil, "/debug")
```

Comment:
Maybe allow this endpoint to be configurable?

Reply:
What is the use case? It is not really different from pprof endpoints, which is configured by default.

Reply:
Hardcoding it to `/debug` is fine.